content/recipes/authoring/kata-snippets/preloaded.md

```diff
@@ -7,4 +7,18 @@ next: /recipes/authoring/kata-snippets/reference-solution/
 
 # Writing Preloaded Code
 
-_This page is a stub._
\ No newline at end of file
+The preloaded code snippet, created by the kata author or translator, can be used to reduce boilerplate code, but when used incorrectly, it can cause lots of problems. To ensure the quality of the kata, it should conform to a set of guidelines presented below.
```

Comment:
Consider choosing 'preloaded code' or '"Preloaded Code"' and sticking with it throughout the guideline (I prefer the first).

Reply:
Done.